scripts/steps/build-api.js

```diff
@@ -110,6 +104,105 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+const CORE_ENTITIES = new Set([
```

Comment:
Add `isFirstOlder`. It is important tool to compare action’s time which is critical for CRDT.

Reply:
Add `parseId`. It is used very often too (to get user ID from action ID).